docs/sphinx/examples/python/advanced_vqe.py

```diff
@@ -29,10 +30,10 @@
 gradient = cudaq.gradients.CentralDifference()
 
 
-def objective_function(parameter_vector: List[float],
+def objective_function(parameter_vector: list[float],
```

Comment:
Note that this will work only in Python 3.9+. We support Python 3.8 as well and it needs `List` from `typing` module.
Consider keeping it so that same code works across all supported Python versions.

Reply:
Good catch, I wasn't even considering that. Reverting back now (I think it was only in this file and maybe 1 or 2 others)